koku/masu/processor/parquet/parquet_report_processor.py

```diff
@@ -370,15 +371,15 @@ def prepare_parquet_s3(self):
 
         if self.provider_type == Provider.PROVIDER_OCP and to_delete:
             # filter the report
-            LOG.warning(log_json(msg="files to delete pre filter", to_delete=to_delete))
+            LOG.info(log_json(msg="files to delete pre filter", to_delete=to_delete))
             to_delete = filter_s3_objects_less_than(
                 self.tracing_id,
                 to_delete,
                 metadata_key="reportnumhours",
-                metadata_value_check=self.metadata_end_date[11:13],
+                metadata_value_check=self.ocp_files_to_process.get(filename.stem).get("meta_reportnumhours"),
                 context=self.error_context,
             )
```

Comment:
nested `get` here can return `None` which lead to errors

Reply:
What is the alternative though? If we get to this point and the `ocp_files_to_process` does not have the filename that we need, what should we do?

Reply:
Might be better to fail on the `KeyError` instead of using `.get()`.

Reply:
I switched this to
```
metadata_value_check=self.ocp_files_to_process[filename.stem]["meta_reportnumhours"],
```